apps/docs/src/app/platform/component-docs/platform-file-uploader/platform-file-uploader-docs.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component } from '@angular/core';
+import { ExampleFile } from '../../../documentation/core-helpers/code-example/example-file';
+
+import * as platformFileUploadDefaultTypesSrc from '!raw-loader!./platform-file-uploader-examples/platform-file-uploader-example.component.html';
+import * as platformFileUploadDefaultTypesTsSrc from '!raw-loader!./platform-file-uploader-examples/platform-file-uploader-example.component.ts';
+import * as platformFileUploadCompactTypesSrc from '!raw-loader!./platform-file-uploader-examples/platform-file-uploader-compact-example.component.html';
+import * as platformFileUploadCompactTypesTsSrc from '!raw-loader!./platform-file-uploader-examples/platform-file-uploader-compact-example.component.ts';
+import * as platformFileUploadMaxFileSizeTypesSrc from '!raw-loader!./platform-file-uploader-examples/platform-file-uploader-max-file-size-example.component.html';
+import * as platformFileUploadMaxFileSizeTypesTsSrc from '!raw-loader!./platform-file-uploader-examples/platform-file-uploader-max-file-size-example.component.ts';
+import * as platformFileUploadMinFileSizeTypesSrc from '!raw-loader!./platform-file-uploader-examples/platform-file-uploader-min-file-size-example.component.html';
+import * as platformFileUploadMinFileSizeTypesTsSrc from '!raw-loader!./platform-file-uploader-examples/platform-file-uploader-min-file-size-example.component.ts';
+import * as platformFileUploadFileTypesSrc from '!raw-loader!./platform-file-uploader-examples/platform-file-uploader-file-types-example.component.html';
+import * as platformFileUploadFileTypesTsSrc from '!raw-loader!./platform-file-uploader-examples/platform-file-uploader-file-types-example.component.ts';
+
+@Component({
+    selector: 'fd-platform-file-uploader-docs',
+    templateUrl: './platform-file-uploader-docs.component.html'
+})
+export class PlatformFileUploaderDocsComponent {
+    defaultFileUploadType: ExampleFile[] = [
+        {
+            language: 'html',
+            code: platformFileUploadDefaultTypesSrc,
+            fileName: 'platform-file-uploader-example'
+        },
+        {
+            language: 'typescript',
+            code: platformFileUploadDefaultTypesTsSrc,
+            fileName: 'platform-file-uploader-example',
+            component: 'PlatformFileUploaderExampleComponent'
+        }
+    ];
+    defaultFileUploadCompactType: ExampleFile[] = [
+        {
+            language: 'html',
+            code: platformFileUploadCompactTypesSrc,
+            fileName: 'platform-file-uploader-compact-example'
+        },
+        {
+            language: 'typescript',
+            code: platformFileUploadCompactTypesTsSrc,
+            fileName: 'platform-file-uploader-compact-example',
+            component: 'PlatformFileUploaderCompactExampleComponent'
+        }
+    ];
+    defaultFileUploadMaxFileSizeType: ExampleFile[] = [
+        {
+            language: 'html',
```

Comment:
for examples that have styles, I think you need to add `scssFileCode` as well so that it reflects correctly in the stackblitz examples